app/src/main/res/layout/activity_registerform.xml

```diff
@@ -85,12 +86,12 @@
         android:id="@+id/backBtn"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="12dp"
-        android:layout_marginBottom="172dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="48dp"
         android:onClick="backBtn_OnClick"
         android:text="Back"
-        app:layout_constraintBottom_toTopOf="@+id/guideline6"
-        app:layout_constraintStart_toStartOf="@+id/guideline3" />
+        app:layout_constraintStart_toStartOf="@+id/guideline3"
```

Comment:
guideline3 is this the rule 3 or what exactly?

Reply:
This is a layout constraint. Placement and size of the components can be based on this.

Reply:
This is the registerform, so you should now better than me I think :). I think guidelines are lines that split the screen in some parts so we can know where we place other things

Reply:
shouldn't we use another name?

Reply:
I think guideline is fine since these are really guidelines